src/changelog/2023/09/custom-node-support.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Custom Node Support
+date: 2023-09-28 12:01:01.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+One feature that was frequently requested was support for custom nodes. Although Node-RED instances today connect automatically to the official Node-RED catalogue, which boasts over 4,000 custom nodes, we understand that specific use cases may require access to additional or different NPM registries and catalogues.
+
+Starting today, you can extend your Node-RED environment by adding custom Node Catalogues. We have introduced the capability to modify the .npmrc file, allowing you to install modules that are not hosted in the official registry.
```

Comment:
This needs documenting on how to achieve it. The feature is implemented like a developer knowing the Node-RED internals would, but I don't know now how to use this.

Reply:
Yes, absolutely, will do

Reply:
I'll write a blog post about this topic

Reply:
https://github.com/FlowFuse/website/pull/1177